numba/cpython/randomimpl.py

```diff
@@ -1952,7 +1952,7 @@ def getitem(a, a_i):
         raise TypeError("np.random.choice() first argument should be "
                         "int or array, got %s" % (a,))
 
-    if size in (None, types.none):
+    if size in (None, types.none) or isinstance(size, types.Omitted):
```

Comment:
Not a proper review, but this ought to be `cgutils.is_nonelike(size)`?

Reply:
Yes, I also notice this API to check it, will refactor.